biz.aQute.remote/src/aQute/remote/agent/AgentServer.java

```diff
@@ -150,19 +150,19 @@ public String uninstall(long... ids) {
 	}
 
 	@Override
-	public String update(Map<String,String> bundles) throws InterruptedException {
+	public String update(LinkedHashMap<String,String> bundles) throws InterruptedException {
```

Comment:
It seems wrong to put an impl specific map type in the API. I understand the implementation will need to use a specific map impl, but the API should not use a specific impl map type.